python/py-sox/Portfile

```
@@ -14,15 +15,18 @@ long_description    ${description}
 
 homepage            https://github.com/rabitt/pysox
 
-checksums           rmd160  e6583059b2e46f28bc76895ccb65f34922fcfbf2 \
-                    sha256  176d3a1e541332a3b581fd60957f684069235fcf61881965f6c9e22e16797a85 \
-                    size    62438
+checksums           rmd160  dcf1bc93b886d7a0a01812dff212926430501673 \
+                    sha256  b0f2d13692450b889cd3f66127e96f801942ec2aac5bb21653dfd150e0d71055 \
+                    size    62423
 
-python.versions     37 38
+python.versions     37 38 39
 
 if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
+    depends_lib-append \
+                    port:sox
```

```suggestion
    depends_run-append \
                    port:sox \
                    port:py${python.version}-numpy
```

I don't think sox is required at build time, so it should be a runtime dep instead.

Also, just looking through the [setup.py](https://github.com/rabitt/pysox/blob/787de8de95d76bdec585d30479ba482c0d444973/setup.py#L27), it seems like NumPy is required.

---

Thanks for all the corrections! Sorry I missed these details in the original PR.